alws/dramatiq/products.py

```diff
@@ -165,10 +165,15 @@ async def set_platform_for_products_repos(
     # name of a product's repo:
     # some-username-some-product-some-platform-x86_64-debug-dr
     repos_per_platform = {
-        f"{product.owner.username}-{product.name}-{platform.name.lower()}-"
-        f"{repo.arch}-{repo_debug_dict[repo.debug]}": platform
+        # fmt: off
+        (
+            f"{product.owner.username}-{product.name}-{platform.name.lower()}"
+            f"-{repo.arch}-{repo_debug_dict[repo.debug]}"
+        ): platform
+        # fmt: on
```

Comment:
do we really need to turn off the black formatter there?
maybe we can create a function that would return the needed string?

Reply:
I think a separate function is overkill

Reply:
and what about using the `join` method then?
I think if you use it, there wouldn't be a need to disable the formatter

Reply:
I removed turning off